src/registry.jl

```diff
@@ -489,3 +489,5 @@ add_format(format"FCS", "FCS", [".fcs"], [:FCSFiles => UUID("d76558cf-badf-52d4-
 add_format(format"HTML", (), [".html", ".htm"], [MimeWriter, SAVE])
 
 add_format(format"MIDI", "MThd", [".mid", ".midi", ".MID"], [:MIDI => UUID("f57c4921-e30c-5f49-b073-3f2f2ada663e")])
+
+add_format(format"QOI", "qoif", ".qoi", [:QOI => UUID("4b34888f-f399-49d4-9bb3-47ed5cae4e65")])
```

Comment:
We're trying to make ImageIO a replacement for ImageMagick and make ImageIO the only interface for normal users, would you mind registering this package to ImageIO instead?

An OpenEXR example can be found in https://github.com/JuliaIO/ImageIO.jl/pull/30 and https://github.com/JuliaIO/FileIO.jl/pull/328.

Reply:
https://github.com/JuliaIO/ImageIO.jl/pull/42, so does this mean it should not be registered in FileIO? I don't immidiately see the benefit of ImageIO over just using FileIO. Should I still have https://github.com/KristofferC/QOI.jl/blob/577645ab3e5930231f4043543d1c622824ceca48/src/QOI.jl#L364-L365 in the package? Should this PR be merged in addition to the ImageIO one, or only ImageIO?

Reply:
> JuliaIO/ImageIO.jl#42, so does this mean it should not be registered in FileIO? I don't immediately see the benefit of ImageIO over just using FileIO.

That's a good question. I'm not sure if my answer satisfies you: the primary goal is to provide an image-related IO toolbox as an alternative to ImageMagick so that when new users come up, we can save the efforts and tell them to `]add ImageIO` instead of `]add PNGFiles QOI ...`.
The latter can sometimes be inconvenient if we have new formats supported. For instance, when QOI.jl is introduced in ImageIO, all ImageIO users automatically get QOI supports.

> Should I still have https://github.com/KristofferC/QOI.jl/blob/577645ab3e5930231f4043543d1c622824ceca48/src/QOI.jl#L364-L365 in the package? Should this PR be merged in addition to the ImageIO one, or only ImageIO?

It depends on how you plan to use or advertise QOI.jl:

- If you like the unified ImageIO way then there's no need to register QOI here.
- If you want people to have `load("img.qoi")` to work with only `]add FileIO QOI`, then you can keep registering QOI here.